### DISCUSSION TITLE
How do I change my Profile Picture

### DISCUSSION THREAD

**Discussion by ItzRealMe** [community/community#58072] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm new and I want to change my profile picture to something else.

**Comment by Thinkright20** [comment, marked as answer]
Follow these steps:

1. Go to your profile
2. Click the big image on the left side of your screen that is currently your profile picture
3. You will be brought to your profile settings, and on the right you have to option to "edit" it
4. You can either upload or remove a photo.

And you should be good! Please mark this as a answer if this helped.

**Reply by ItzRealMe** [reply]
Ok thanks
